### ISSUE TITLE
Video is not inserted after being recorded (Chrome only)

### ISSUE THREAD

**Issue by felipebochehin87** [google/web-stories-wp#11912]
## Bug Description

In Chrome, the video recording works, but when I click on Insert I see error in the console and the video is not uploaded. It does work in Firefox and Safari.

## Expected Behaviour

Video should be uploaded when user clicks on "Insert".

## Steps to Reproduce

1. Use PR app 11887 or 11908
2. Start a new story
3. Click on the background
4. Click on Record Video/Audio
5. Once record is complete, click on Insert

## Screenshots

[https://app.zenhub.com/files/235435637/b746d6bc-b480-47fb-aa36-aa4c8602c9e1/download](https://app.zenhub.com/files/235435637/b746d6bc-b480-47fb-aa36-aa4c8602c9e1/download)

https://images.zenhubusercontent.com/235435637/e13f9a21-5d79-4114-9019-1fa8d9088628/2022_07_13_19_28_39.mp4

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System: Windows 10 64 bit
- Browser: Chrome

**Comment by felipebochehin87** [comment]
@swissspidy just saw your comment... Ok. I'll try to dig more to understand why this happens here... TY!
